<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modelagem ArboBA</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(33, 37, 41);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="arboVirose">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Dengue</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Chikungunya</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Zika</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="selecao">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>100</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>60</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Seleção:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="confirmarArbo">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>100</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>false</bold>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(27, 135, 85);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Confirmar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="confirmarResto">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>100</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>false</bold>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(27, 135, 85);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Confirmar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Arbovirose:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="periodo">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>100</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(108, 117, 125);</string>
    </property>
    <property name="text">
     <string>Periodo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="avancar">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>740</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>false</bold>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(27, 135, 85);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Prosseguir</string>
    </property>
   </widget>
   <widget class="QTableView" name="dataframe">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>901</width>
      <height>541</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(108, 117, 125);</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-10</y>
      <width>1021</width>
      <height>831</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <zorder>avancar</zorder>
   <zorder>dataframe</zorder>
   <zorder>frame</zorder>
   <zorder>label_2</zorder>
   <zorder>confirmarArbo</zorder>
   <zorder>confirmarResto</zorder>
   <zorder>label</zorder>
   <zorder>selecao</zorder>
   <zorder>arboVirose</zorder>
   <zorder>periodo</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
